scripts/pdf/README.md

```diff
@@ -27,6 +27,10 @@ Generating the PDF is as simple as running:
 
     python3 render.py <path-to-pages-directory> --color <color-scheme>
```

Comment:
I'd get rid of the following example, and replace this with:

```suggestion
    python3 render.py <path-to-pages-directory> [--color <color-scheme>] [--output <filename>]
```

with maybe a bit underneath that it defaults to the `basic` color and `tldr-pages.pdf` as the output filename. Alternatively get rid of the example altogether as well as the bit about available colors and just have people run `python3 render.py --help` themselves, or just copy its output here.

Reply:
Thanks for the review, I will try to update the logic to use `basic` as default too along with your other suggestions.